migrations/v0.7/README.md

```diff
@@ -0,0 +1,13 @@
+# v0.7 migration
+
+_Please note: this notice applies when upgrading to the latest v0.7.x release._
+
+When migrating from Vela version [v0.6.0](https://github.com/go-vela/community/blob/master/releases/v0.6.0.md) to [v0.7.2](https://github.com/go-vela/community/blob/master/releases/v0.7.2.md) the Vela administrator will want to ensure the following actions are being performed:
+
+1. Update tables in the Postgres database
+   * `ALTER TABLE users ADD COLUMN IF NOT EXISTS refresh_token VARCHAR(500);`
+   * `ALTER TABLE builds ADD COLUMN IF NOT EXISTS deploy_payload VARCHAR(2000);`
```

Comment:
Need to also add `ALTER TABLE workers ADD COLUMN IF NOT EXISTS build_limit INTEGER;`.

Source: https://github.com/go-vela/server/pull/219

Reply:
added. thanks!